### PR TITLE
Adding review state (closes #23)

### LINKED ISSUE #23: Be able to mark as needing review again

### PR DESCRIPTION
**Pull request by escowles**

### CODE REVIEW

File: app/views/curation_concerns/base/_form_state.html.erb [hunk 1]
```diff
@@ -2,11 +2,16 @@
   <legend>State</legend>
   <div class="form-group">
```
Comment (tpendragon) [inline app/views/curation_concerns/base/_form_state.html.erb]: Can simple form help here to reduce some of the logic?

Comment (escowles) [inline app/views/curation_concerns/base/_form_state.html.erb]: I can generate the radio buttons pretty easily, e.g.:

```
  <% states = { pending: 'In progress and suppressed from public display',
                review: 'In need of review',
                complete: 'Finished work, visibility determined by Access Rights' } %>
  <%= f.input :state, collection: states.map {|k,v| [k.to_s.titleize + ": " + v,k]}, as: :radio_buttons %>
```

But how do we generate the badges?

Comment (jpstroop) [inline app/views/curation_concerns/base/_form_state.html.erb]: This seems pretty clean to me; what logic is of concern?

Comment (tpendragon) [inline app/views/curation_concerns/base/_form_state.html.erb]: Just concerning that we have to update the checked value postfix conditional based on state for each state there is, which I thought simple form might be able to handle.

Comment (tpendragon) [inline app/views/curation_concerns/base/_form_state.html.erb]: > But how do we generate the badges?

That's a good point. I'd have to fiddle with it, might not be able to.

Comment (tpendragon) [inline app/views/curation_concerns/base/_form_state.html.erb]: Maybe as simple as a pending? method?

**Pull request merged**